File: 2535-difference-between-element-sum-and-digit-sum-of-an-array/2535-difference-between-element-sum-and-digit-sum-of-an-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var differenceOfSum = function(nums) {
    let elementSum = 0;
    let digitSum = 0;
    let digitValue = nums.join("").split("");
    
    for(let i = 0; i < nums.length; i++) {
        elementSum += nums[i];
    }
    
    for(let j = 0; j < digitValue.length; j++) {
        digitSum += Number(digitValue[j]);
    }
    return Math.abs(elementSum - digitSum);
};

File: 2535-difference-between-element-sum-and-digit-sum-of-an-array/2535-difference-between-element-sum-and-digit-sum-of-an-array.py
class Solution:
    def differenceOfSum(self, nums: List[int]) -> int:
        def digit_sum(num):
            return sum(int(digit) for digit in str(num) if digit.isdigit())
        
        element_sum = sum(nums)
        digit_sum = sum(digit_sum(num) for num in nums)
        
        return abs(element_sum - digit_sum)
        